pkg/executor/distsql.go

```diff
@@ -1168,7 +1168,11 @@ func (w *tableWorker) pickAndExecTask(ctx context.Context) {
 	defer func() {
 		if r := recover(); r != nil {
 			logutil.Logger(ctx).Error("tableWorker in IndexLookUpExecutor panicked", zap.Any("recover", r), zap.Stack("stack"))
-			task.doneCh <- errors.Errorf("%v", r)
+			err := errors.Errorf("%v", r)
```

Comment:
why make this modification?

Reply:
*error.fundamental is not a *error.Error, so we can't keep the error code, or check the error is the same type...